starter-code/index.js

```diff
@@ -1,11 +1,55 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
+    constructor(items, length) {
+      this.items = [];
+      this.length = 0;
+    }
+    add(item) {
+      this.items.push(item);
+      this.items.sort(function(a,b){
+        return a-b;
+      })
+      this.length += 1;
+    }
+    get(pos) {
+      if (pos <= this.length) {
+        return this.items[pos];
+      } else {
+        throw new Error("OutOfBounds");
+      }
+    }
+    max() {
+      if (this.length === 0) {
+        throw new Error("EmptySortedList");
+      } else {
+        return Math.max(...this.items);
+      }      
+    }
+    min() {
+      if (this.length === 0) {
+        throw new Error("EmptySortedList")
+      } else {
+        return Math.min(...this.items);
+      } 
+    }
+    sum() {
+      let somme = 0;
+      if (this.length === 0) {
+        return 0;
+      } else {
+        for (let i=0 ; i<this.length ; i++) {
+          somme = somme + this.items[i];
+        }
+        return somme;
+      }      
```

Comment:
Et sur ta fonction sum() tu pouvais utiliser reduce : 
sum() {
    return this.items.reduce((acc, item) => acc + item, 0);
  }